phantomgames/Search/SearchGameViewController.swift

```diff
@@ -13,22 +13,30 @@ class SearchGameViewController: UIViewController {
     
     @IBOutlet weak private var tableView: UITableView!
     private lazy var viewModel = SearchGameViewModel(repository: SearchGameRepository(), delegate: self)
+
     // MARK: UI Component
     private let searchController = UISearchController(searchResultsController: nil)
     
+    var gamesURL: String?
+    
     // MARK: Functions
     
     override func viewDidLoad() {
         super.viewDidLoad()
         setupTableView()
         setupSearchController()
-        viewModel.fetchSearchResults()
+        if let url = gamesURL {
+            viewModel.fetchSearchResults(fromURL: url)
+        } else {
+            viewModel.fetchSearchResults()
+        }
```

Comment:
I'm not dead sure at the moment if the one above and this will complain, but it shouldn't

If it does just resolve this comment

```suggestion
        if let gamesURL {
            viewModel.fetchSearchResults(fromURL: gamesURL)
        } else {
            viewModel.fetchSearchResults()
        }
```